### ISSUE TITLE
[BUG]validation loss not printed when using slightly modified hyperparameters

### ISSUE THREAD

**Issue by saraswat40** [autogluon/autogluon#3247]
I am following this tutorial: https://auto.gluon.ai/dev/tutorials/tabular/advanced/tabular-multilabel.html

When I run training like so:
`multi_predictor.fit(train_df_with_labels, tuning_data=valid_df, ag_args_fit={'num_gpus': 1})
`
The validation loss is printed like so:
```
Fitting TabularPredictor for label: GO:0005575 ...
[LightGBM] [Fatal] GPU Tree Learner was not enabled in this build.
Please recompile with CMake option -DUSE_GPU=1
[1000]	valid_set's binary_logloss: 0.529833
[2000]	valid_set's binary_logloss: 0.515588
[3000]	valid_set's binary_logloss: 0.508876
[4000]	valid_set's binary_logloss: 0.506015
[5000]	valid_set's binary_logloss: 0.506222
```

Because of [this](https://github.com/autogluon/autogluon/issues/3246) and [this](https://github.com/autogluon/autogluon/issues/520) I tried modifying the hyperparameters and exclude LIghtGBM. My output looks like this now:
`Fitting TabularPredictor for label: GO:0005575` ...

No validation loss is printed. Let me know if I missed something.

**Comment by Innixma** [comment]
The loss is only printed for LightGBM in this case, so it won't print if you exclude LightGBM. You can get other model's losses via setting `verbosity` to 3 or 4.
